libosdp-sys/src/lib.rs

```diff
@@ -8,4 +9,4 @@
 #![allow(missing_docs)]
 #![allow(unused)]
 
-include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+core::include!(core::concat!(core::env!("OUT_DIR"), "/bindings.rs"));
```

Comment:
This hunk seems to be missing from PR #16. Let me create a fixup patch locally so a rebase on this branch would drop changes to this file.

Reply:
I just restored the version and set `libosdp-sys` to be a path dependency.